File: .ai/runtime/backups/3.27.6/.ai/managed/qualification/fixtures/java/src/test/java/CalcTest.java
import org.junit.jupiter.api.Test;
import static org.junit.jupiter.api.Assertions.assertEquals;

public class CalcTest {
    @Test void testAdd() { assertEquals(5, Calc.add(2, 3)); }
    // Заведомо падающий baseline-тест: харнесс проверяет, что движок извлекает Class.method
    // упавшего теста из surefire-вывода (CalcTest.testSub), а не пустое множество.
    @Test void testSub() { assertEquals(999, Calc.sub(5, 2)); }
}

File: .ai/runtime/backups/3.27.6/.ai/managed/qualification/fixtures/go/calc_test.go
package calcfixture

import "testing"

func TestAdd(t *testing.T) {
	if Add(2, 3) != 5 {
		t.Errorf("Add(2,3) = %d; want 5", Add(2, 3))
	}
}

func TestSub(t *testing.T) {
	// Заведомо падающий baseline-тест: харнесс снимает РЕАЛЬНЫЙ вывод `go test` и проверяет,
	// что движок извлекает имя упавшего теста (--- FAIL: TestSub), а не мусорный {'FAIL'}.
	if Sub(5, 2) != 999 {
		t.Errorf("Sub(5,2) = %d; want 999", Sub(5, 2))
	}
}
